_includes/page__hero.html

```diff
@@ -2,11 +2,6 @@
 
 <div class="page__hero--overlay home_hero" style="background: {{ page.header.overlay_color | default: 'white' }}">
     <div class="background_media" style="{% if page.header.image %}background-image: url('{{page.header.image | relative_url}}'){% endif %}">
-        {% if page.header.video %}
```

Comment:
Ideally this video should be removed only for small resolutions

Reply:
This can be removed in the ticket #46 to improve the performance at mobiles